src/bumper-responsive-image.coffee

```diff
@@ -62,6 +95,28 @@
 
       return fullUrl
 
+    # Gets the full url based on bumper data attributes
+    # @param el [HTML Element] html img or div element that has responsive image data attributes
+    # @param breakpoint [String] an optional name of a breakpoint (as defined from setBreakpoints)
+    #
+    getUrl: (el, breakpoint = 'default') ->
+      url = el.getAttribute("data-bumper-responsive-image-url-#{breakpoint}") ||
+            el.getAttribute('data-bumper-responsive-image-url')
+      params = el.getAttribute("data-bumper-responsive-image-url-params-#{breakpoint}") ||
+               el.getAttribute('data-bumper-responsive-image-url-params')
+
+      # Log warning if no url is defined
+      throw new Error "data-bumper-responsive-image-url[-#{breakpoint}] is not set." unless url
+
+      # combine params if they are found
+      fullUrl = if params then "#{url}?#{params}" else url
+
+      # detect if inteprolation is needed
+      if window.Bumper.Dom? && fullUrl.indexOf('{') >= 0
+        fullUrl = window.Bumper.Dom.interpolateElementAttrs fullUrl, el, @options.restrictToParents
```

Comment:
@mmoriar1 and this i where its called.  you can set it via `window.Bumper.Responsive.Image. restrictToParents = true`